app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find_by(id: params[:id])
+  end
+end
```

Comment:
- [ ] `index` action is missing in the _posts_ controller. Kindly ensure that both `index` and `show` actions are present for the posts controller so as to achieve corresponding _views_ similar to the given [wireframe template](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md).